embedded/sql/stmt.go

```diff
@@ -1941,6 +2330,50 @@ func (bexp *BinBoolExp) reduceSelectors(row *Row, implicitDB, implicitTable stri
 	}
 }
 
+func (bexp *BinBoolExp) isConstant() bool {
+	return bexp.left.isConstant() && bexp.right.isConstant()
+}
+
+func (bexp *BinBoolExp) selectorRanges(table *Table, params map[string]interface{}, rangesByColID map[uint64]*typedValueRange) error {
+	if bexp.op == AND {
+		err := bexp.left.selectorRanges(table, params, rangesByColID)
```

Comment:
We could merge ranges from left and right, right? Do you leave this for improvements in the future?

Could be useful when we've got queries like:
`SELECT * FROM person WHERE age > 10 AND age < 20`

Reply:
this is actually happening with current implementation

Reply:
True, I misunderstood the code